### ISSUE TITLE
Maintainers

### ISSUE THREAD

**Issue by blakeembrey** [typings/registry#70]
If you're interested in helping out to merge type definitions, please let me know :smile: I'd love to add some collaborators to the registry to merge definitions and offer helpful advice.

**Comment by myitcv** [comment]
@blakeembrey - very happy to help. I've been focussed on other things the last couple of months, but will be focussing on our use of typings again in the next couple of weeks.

**Comment by blakeembrey** [comment]
Thanks :+1: I've given you access to the registry. The only thing to keep in mind is that the PRs should pass and I've been doing a little reviewing on PRs in case people have been misunderstanding pieces. The test suite is quite simple, just checks the new files added in git are valid and that the versions install with typings correctly. There's an indexer every 10 minutes adding them to the public API.

**Comment by dsebastien** [comment]
Sure, why not :)

**Comment by blakeembrey** [comment]
Awesome, cheers :+1: Added.

**Comment by unional** [comment]
Yes, I would love to.  I'm going to create a yo generator for bootstrap typings projects. WIll be ready in a few hours.

**Comment by blakeembrey** [comment]
@unional Good idea. I was going to make a "project" command with the same idea, but `yo` would work. Ref: https://github.com/typings/typings/issues/118

**Comment by http91** [comment]
I'm not available right now, gotta sharpen my skills, but I'd like to help in the future. I wanted to show my support even though I'm unable to help right now.

**Comment by samvloeberghs** [comment]
Sure :+1:

**Comment by dvidsilva** [comment]
I'll be glad to help too, wanna see the angular definitions happening.

**Comment by Fank** [comment]
@dvidsilva Angular 1 or 2?

**Comment by dvidsilva** [comment]
angular 1, 1.5 mostly. I'm using the --ambient flag for now.

**Comment by heruan** [comment]
I'll be glad to help!

**Comment by blakeembrey** [comment]
How would you guys like to help most? Writing definitions can be done independently, but we can move the repo into @typed-typings when you're ready 😄

**Comment by heruan** [comment]
I would help merging definitions and take on requests to fulfill them, plus being available to any other collaboration to the project which I really believe in!

**Comment by johnnyreilly** [comment]
I like this guy 😀👍

**Comment by unional** [comment]
:+1: :clap:

**Comment by felixfbecker** [comment]
I would like to help out. Already did a lot of work for [typed-sequelize](/louy/typed-sequelize), wrote [typed-bignumber.js](/felixfbecker/typed-bignumber.js) and currently finishing [typed-express](/felixfbecker/typed-express)

**Comment by blakeembrey** [comment]
@felixfbecker Awesome! I've invited you to the registry, and @types. Feel free to move some repositories into @types, I would be happy to help out with Express.

**Comment by beenotung** [comment]
I see there are many open issues, let me help out as well :smile:

**Comment by blakeembrey** [comment]
@beenotung The best help would be getting some of the issues closed, I think most issues in the registry are just requests. However, anyone that wants to move repos into @types is welcome to as well, we just need to make sure there's someone going to maintain the repo 😄
